src/crypto/crypto_context.h

```diff
@@ -43,13 +44,28 @@ class SecureContext final : public BaseObject {
 
   const SSLCtxPointer& ctx() const { return ctx_; }
 
+  // Non-const ctx() that allows for non-default initialization of
+  // the SecureContext.
+  SSLCtxPointer& ctx() { return ctx_; }
+
   SSLPointer CreateSSL();
 
   void SetGetSessionCallback(GetSessionCb cb);
   void SetKeylogCallback(KeylogCb cb);
   void SetNewSessionCallback(NewSessionCb cb);
   void SetSelectSNIContextCallback(SelectSNIContextCb cb);
 
+  inline const X509Pointer& issuer() const { return issuer_; }
+  inline const X509Pointer& cert() const { return cert_; }
+
+  MUST_USE_RESULT v8::Maybe<bool> AddCert(Environment* env, BIOPointer&& bio);
+  MUST_USE_RESULT v8::Maybe<bool> SetCRL(Environment* env,
+                                         const BIOPointer& bio);
```

Comment:
It confuses me that these two new functions are both marked with `MUST_USE_RESULT`, but all call sites that are added in this PR silently discard the respective results.